components/Header.tsx

```diff
@@ -5,21 +5,14 @@ import { Text, View } from '../components/Themed';
 import SVGImage from '../assets/svgs/splash.svg';
 
 
-export default function Header() {
+export default function Header(props:any) {
 	return (
-		<View style={styles.container}>
-			<SVGImage width={150} height={75} />
+		<View style={{backgroundColor:props.backgroundColor, paddingTop:60, paddingLeft:24}}  >
+			<SVGImage  width={150} height={75} />
 		</View>
 	);
 }
 
 const styles = StyleSheet.create({
```

Comment:
remove you're not using

Reply:
That is the logo, added a background color prop because the default background color is light gray and side menu background is white

Reply:
i'm talking about line 16, can see anything in the object

Reply:
ok